labs/b4.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Beginner Lab 4 - Debian, packages, compiling software
+layout: lab
+---
+
+## What is a distribution?
+
+A Linux distribution is a combination of the Linux kernel, package management system, drivers, scheduler, and other software. They are the operating systems that use Linux as a kernel. 
+
+## What should I use then?
+
+Currently, there are around 300 actively developed distributions listed on DistroWatch. It might be a bit overwhelming with so many choices. Major distributions include Debian, Arch Linux, and Red Hat Linux, but in this course, we’ll only be using Debian.
```

Comment:
I think adding something here about using distros that are more common or more supported is good, then they have more packages available, community support, potential commercial support, more experience with other people using them, etc.